dsperse/src/server/handlers.py

```
@@ -53,99 +55,100 @@ async def handle_start_incremental_run(ctx: ServerContext, req: dict) -> dict:
 
-    with tempfile.TemporaryDirectory() as tmp_dir:
+    data_dir = RUN_DATA_DIR / run.run_uid
+    data_dir.mkdir(parents=True, exist_ok=True)
+
+    asyncio.ensure_future(_process_run(ctx, run, inputs, slices_dir, data_dir))
+    print(f"[start_incremental_run] run_uid={run.run_uid}, data_dir={data_dir}", flush=True)
+
+    return {"run_uid": run.run_uid, "status": run.status.value}
+
+
+async def _process_run(ctx: ServerContext, run, inputs, slices_dir: Path, data_dir: Path):
```

Ruff (0.15.1)

[warning] Unused function argument: `ctx`

(ARG001)